test/TensorFlowRuntime/tensor.swift

```diff
@@ -276,6 +276,26 @@ TensorTests.testAllBackends("SimpleMath") {
                              byError: 0.0001)
 }
 
+TensorTests.testAllBackends("StandardDeviation") {
+  expectEqual(0, Tensor<Float>([1]).standardDeviation().scalarized())
+  expectEqual(
+      0.5,
```

Comment:
The indentation here should be 2. In the standard library code base, all line breaks are indented by 2.